applications/StructuralMechanicsApplication/tests/test_dynamic_eigenvalue_analysis.py

```diff
@@ -0,0 +1,159 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.EigenSolversApplication as EigenSolversApplication
```

Comment:
this will not pass travis since there this EigenSolversApp is not compiled
=> use skiptest in case this is not available

Reply:
Done. Please check if this is correct.